docs/user/02-logs.md

```diff
@@ -160,7 +160,8 @@ spec:
     ...
```

Comment:
There is also a CAUTION note in Step 1.

Reply:
I ignored it because it's indented and wouldn't render correctly. It belongs to the option "custom" :/

Reply:
fixed with https://github.com/kyma-project/telemetry-manager/pull/792/commits/3d7c4ecadc9855d8947a957e779e17d493d15339